docs/README.md

```diff
@@ -1,27 +1,102 @@
+        </tr>
+        <tr>
+            <td align="center" width="420"><em>High-quality up-to-date documentation</em></td>
+            <td align="center">→</td>
+            <td align="center" width="420"><strong><em>Trust and reliability</em></strong></td>
+        </tr>
+    </table>
+</div>
+
+<p align="center">
+    <a href="https://github.com/Mirascope/mirascope/actions/workflows/tests.yml" target="_blank"><img src="https://github.com/Mirascope/mirascope/actions/workflows/tests.yml/badge.svg?branch=main" alt="Tests"/></a>
+    <a href="https://docs.mirascope.io/" target="_blank"><img src="https://img.shields.io/badge/docs-available-brightgreen" alt="Docs"/></a>
+    <a href="https://pypi.python.org/pypi/mirascope" target="_blank"><img src="https://img.shields.io/pypi/v/mirascope.svg" alt="PyPI Version"/></a>
+    <a href="https://github.com/Mirascope/mirascope/stargazers" target="_blank"><img src="https://img.shields.io/github/stars/Mirascope/mirascope.svg" alt="Stars"/></a>
+</p>
 
 ---
 
-[![GitHub stars](https://img.shields.io/github/stars/Mirascope/mirascope.svg)](https://github.com/Mirascope/mirascope/stargazers)
-[![Documentation](https://img.shields.io/badge/docs-available-brightgreen)](https://docs.mirascope.io/)
-[![](https://github.com/Mirascope/mirascope/actions/workflows/tests.yml/badge.svg?branch=main)](https://github.com/Mirascope/mirascope/actions/workflows/tests.yml)
-[![GitHub issues](https://img.shields.io/github/issues/Mirascope/mirascope.svg)](https://github.com/Mirascope/mirascope/issues)
-[![Github discussions](https://img.shields.io/github/discussions/Mirascope/mirascope)](https:github.com/Mirascope/mirascope/discussions)
-[![GitHub license](https://img.shields.io/github/license/Mirascope/mirascope.svg)](https://github.com/Mirascope/mirascope/blob/main/LICENSE)
-[![PyPI version](https://img.shields.io/pypi/v/mirascope.svg)](https://pypi.python.org/pypi/mirascope)
-[![PyPI pyversions](https://img.shields.io/pypi/pyversions/mirascope.svg)](https://pypi.python.org/pypi/mirascope)
+**Documentation**: <a href="https://docs.mirascope.io" target="_blank">https://docs.mirascope.io</a>
+
+**Source Code**: <a href="https://github.com/Mirascope/mirascope" target="_blank">https://github.com/Mirascope/mirascope</a>
 
 ---
 
+**Mirascope** is a library purpose-built for Prompt Engineering on top of <a href="https://pydantic.dev" target="_blank">Pydantic</a>:
+
+```python
+from mirascope import Prompt, messages
+
+
+@messages
+class BookRecommendationPrompt(Prompt):
+    """
+    SYSTEM:
+    You are the world's greatest librarian.
+
+    USER:
+    I've recently read the following books: {titles_in_quotes}.
+    What should I read next?
+    """
+
+    book_titles: list[str]
+
+    @property
+    def titles_in_quotes(self) -> str:
+        """Returns a comma separated list of book titles each in quotes."""
+        return ", ".join([f'"{title}"' for title in self.book_titles])
```

Comment:
since this is the very first thing someone sees when looking at mirascope, I think it might be worth leaving out the messages decorator/system message part of this prompt so the first example really hones in on the pydantic aspect

Reply:
Done.